### DISCUSSION TITLE
Neues Flashen für Release V23.6.9 notwendig?

### DISCUSSION THREAD

**Discussion by OpenShorty** [tbnobody/OpenDTU#1024] [Q&A] [answered]
Kurze Frage zum neuen Release V23.6.9 .
Ich habe meine DTU wie anfänglich beschrieben mit der Partitionstabelle geflasht. Also mit den 4 .bin Dateien.
In der Anleitung mit dem Release V23.6.9 hat sich ja die Anleitung geändert.

Muss ich also meine DTU nun mit nur einer Datei neu flashen, bevor ich die V23.6.9 aufspiele?

**Comment by Solarteur** [comment]
Auf welche Version soll Release V23.6.9 neu aufgespielt werden?

**Comment by OpenShorty** [comment]
Ich habe die vorherige V23.6.1 drauf.  Wie gesagt nur das Flashen mit den 4 .bin Dateien gemacht.

**Reply by Solarteur** [reply]
Ich habe das Update von V23.6.1 auf V26.6.9 über OTA gemacht. 
Einstellungen --> Firmware-Aktualisierung und dort die aktuelle "opendtu-generic.bin" auswählen - fertig.

**Reply by OpenShorty** [reply]
Ja so habe ich die vorherigen Updates auch gemacht. Nur mit dieser Version hat sich ha die Partitionstabelle geändert.
Vorher geflasht mit 

Adresse | Datei


0x1000 | bootloader.bin
0x8000 | partitions.bin
0xe000 | boot_app0.bin
0x10000 | opendtu-*.bin

in der neuen steht nur noch

Adresse | Datei
0x0 I opendtu-generic.factory.bin.

Da ist meine Frage ob ich die Partitionstabelle auch erneuern muss oder ob das mit den 4 .bin Dateien auch geht.

**Reply by tbnobody** [reply]
Das was du vorher mit den 4 Dateien gemacht hast kannst du jetzt einfach mit der .factory Datei an Adresse 0x0 machen. Das der Einsatzzweck der opendtu-generic.bin bleibt wie gehabt.

Wenn man vorher was mit 4 Dateien gemacht hat muss man jetzt auch nicht alles platt machen. Es wurden nur die 4 Dateien in eine konvertiert.

**Comment by Solarteur** [comment]
Ja, mit V23.6.1 gab es noch die "alte" Partitionstabelle. Wie schon geschrieben: Die "opendtu-generic.bin" (**nicht** "opendtu-generic.**factory**.bin") auswählen und updaten - hat bei mir ohne Probleme geklappt.

**Reply by OpenShorty** [reply]
Okay also hast du nicht neu geflasht.
Meine Frage zielte nur darauf hin ob man neu Flashen sollte , wenn man vorher mit den 4 Dateien geflasht hat.
Ob es Sinn mach, bzw. ob es Vorteile hat wenn man mit der neuen Datei neu flasht oder Nachteile hat wenn man die geflashten 4 Dateien beibehält.

**Comment by tbnobody** [comment, marked as answer]
> Muss ich also meine DTU nun mit nur einer Datei neu flashen, bevor ich die V23.6.9 aufspiele?

Wenn das so wäre hätte ich das wo hingeschrieben :)
Alles gut, die 4 Dateien wurden nur zu einer konvertiert. Außer deine alte Version ist von vor dem 15.03.2023 und du hast noch die alte Partitionstabelle.

**Reply by OpenShorty** [reply]
Super vielen Dank für die Antwort. 

Es wäre vllt. hilfreich für Neulinge auch die Readme mit den !! IMPORTANT UPGRADE NOTES !! anzupassen. Da steht noch das alte drin und unten steht es dann anders. Wenn man sich nicht ganz sicher ist kann das verwirren.

**Reply by tbnobody** [reply]
Du hast recht, aber Neulinge haben dann auch kein bestehendes OpenDTU von März. Ich werde aber den Upgrade Guide noch anpassen.
